src/app/app-routing.module.ts

```diff
@@ -7,79 +7,79 @@ import { CancelEditTechGuard } from '@guards/cancel-edit-tech/cancel-edit-tech.g
 import { FeatureToggleGuard } from '@guards/feature-toggle-guard/feature-toggle.guard';
 import { RoleGuard } from '@guards/role-guard/roles.guard';
 import { Roles } from '@models/roles.enum';
+import { RootRoutes } from '@models/routes.enum';
 import { techRecordViewResolver } from './resolvers/tech-record-view/tech-record-view.resolver';
 import { titleResolver } from './resolvers/title/title.resolver';
 
 const routes: Routes = [
   {
-    path: '',
+    path: RootRoutes.ROOT_ROUTE_ROOT,
     resolve: { title: titleResolver },
     children: [
       {
-        path: '',
+        path: RootRoutes.ROOT_ROUTE_ROOT,
```

Comment:
We need to look at the name here, RootRoutes.ROOT_ROUTE_ROOT

Surely the same info is contained in something like `RootRoutes.ROOT` etc

Reply:
I am wondering if it ok to leave '' to the root path, as it not a magic string. I try to replace as enum pattern here, but I am prefer leave it.

Reply:
Yeah, I will amend it now.

Reply:
agree with this, looking at the enum file, all the route names can lose the prefix as the enum name is the same